ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -222,6 +222,11 @@
     reason="Test runs ONLY on managed service consumer cluster",
 )
 
+ipi_or_managed_deployment_required = pytest.mark.skipif(
```

Comment:
Since there is no tight connection between IPI and managed service, please split into 2 decorators - one for IPI and one for managed service